blocksec2go/commands.py

```diff
@@ -4,6 +4,10 @@
 from smartcard.System import readers
 from blocksec2go.comm.pyscard import open_pyscard
 
+from cryptography.hazmat.primitives.asymmetric import ec
```

Comment:
When using an external library please add it to setup.py so that i.e. "pip install"... automatically resolves the dependencies